src/Restaurant.js

```diff
@@ -1,13 +1,29 @@
-import React from "react";
-
+import React, { useState } from "react";
```

Comment:
```
import React from "react";
```

The useState import has moved to a different file again here. This isn't a massive problem, but you will see warning in your console!